### DISCUSSION TITLE
Running emulators on v86

### DISCUSSION THREAD

**Discussion by hello-smile6** [copy/v86#585] [Q&A] [answered]
@copy Would it be possible to run a 64-bit VM within v86?

**Comment by copy** [comment, marked as answer]
Sure, why not?

**Reply by hello-smile6** [reply]
Not sure. Could you help me install a modern CLI web browser on Ubuntu 10? (e.g. elinks) `apt-get` is useless, and dpkg won't work at all unless I can download a .deb .
<h6><details>
<summary></summary>
@JaydenDev
</details></h6>

**Reply by jdev082** [reply]
He's trying to install my web browser (minimum ubuntu version supported is 14.04) on Ubuntu 10 which very much won't have the correct packages for installation an run time.

**Reply by jdev082** [reply]
Which requires a few possibly unavailable libraries for such ubuntu version.

**Reply by hello-smile6** [reply]
> He's trying to install my web browser (minimum ubuntu version supported is 14.04) on Ubuntu 10 which very much won't have the correct packages for installation an run time.

Not trying to install yours. Trying to install elinks. Here's a dump of /usr/bin
<details><summary>/usr/bin</summary>
<pre>
[
2to3
2to3-2.6
a2p
aconnect
acpi_fakekey
acpi_listen
add-apt-repository
addpart
addr2line
alacarte
alsamixer
amidi
amixer
amuFormat.sh
animate
aplay
aplaymidi
apport-bug
apport-cli
apport-collect
apport-unpack
appres
apropos
apt-add-repository
apt-cache
apt-cdrom
apt-config
aptdcon
apt-extracttemplates
apt-ftparchive
apt-get
aptitude
aptitude-create-state-bundle
aptitude-run-state-bundle
apt-key
apt-mark
apt-sortpkgs
apturl
apturl-gtk
ar
arch
arecord
arecordmidi
arm2hpdl
arping
as
aseqdump
aseqnet
aspell
aspell-import
at
atobm
atq
atrm
avahi-browse
avahi-browse-domains
avahi-publish
avahi-publish-address
avahi-publish-service
avahi-resolve
avahi-resolve-address
avahi-resolve-host-name
avahi-set-host-name
awk
baobab
base64
basename
bashbug
batch
bc
bdftopcf
bdftops
bdftruncate
bf_compact
bf_compact-bdb
bf_copy
bf_copy-bdb
bf_tar
bf_tar-bdb
bitmap
bluetooth-agent
bluetooth-applet
bluetooth-properties
bluetooth-sendto
bluetooth-wizard
bmtoa
bogofilter
bogofilter-bdb
bogolexer
bogolexer-bdb
bogotune
bogotune-bdb
bogoupgrade
bogoupgrade-bdb
bogoutil
bogoutil-bdb
brasero
bsd-write
btcflash
byobu
byobu-config
byobu-export
byobu-janitor
byobu-launch
byobu-launcher
byobu-launcher-install
byobu-launcher-uninstall
byobu-reconnect-sockets
byobu-select-profile
byobu-select-session
byobu-status
byobu-status-detail
c2ph
c89
c89-gcc
c99
c99-gcc
cal
calendar
calibrate_ppa
canberra-gtk-play
cancel
captoinfo
casper-reconfigure
catchsegv
catman
cautious-launcher
cc
cdparanoia
cdrecord
c++filt
chage
charmap
chattr
chcon
check-bios-nx
checkbox-gtk
check-language-support
checksctp
chfn
chkdupexe
chrt
chsh
ciptool
ckbcomp
ck-history
ck-launch-session
ck-list-sessions
cksum
clear
clear_console
cli
cli-gacutil
cmp
codepage
col
colcrt
colrm
column
comm
compare
compiz
compiz-decorator
compose
composite
config_data
conjure
convert
corelist
couchdb
couchdb-dump
couchdb-load
couchjs
couchpy
cpan
cpan2dist
cpanp
cpanp-run-perl
cpp
cpp-4.4
cpufreq-selector
c_rehash
crontab
csplit
csslint-0.6
csv2vcard
ctstat
cups-calibrate
cupstestdsc
cupstestppd
cut
cvt
dbus-launch
dbus-monitor
dbus-send
dc
dccleancrw
dcfujigreen
dcfujiturn
dcfujiturn16
dcparse
dcraw
ddate
deallocvt
debconf
debconf-apt-progress
debconf-communicate
debconf-copydb
debconf-escape
debconf-set-selections
debconf-show
defoma
defoma-app
defoma-font
defoma-hints
defoma-id
defoma-psfont-installer
defoma-subst
defoma-user
delpart
desktop-file-install
desktop-file-validate
devdump
dexconf
dfutool
dh_bash-completion
dh_installdefoma
dh_installxmlcatalogs
dh_pycentral
dh_pysupport
diff
diff3
dig
dircolors
directomatic
dirname
dirsplit
display
do-release-upgrade
dotlockfile
dpkg
dpkg-deb
dpkg-divert
dpkg-query
dpkg-split
dpkg-statoverride
dpkg-trigger
dprofpp
du
dumphint
dumpkeys
dvd-ram-control
dvd+rw-booktype
dvd+rw-format
dvd+rw-mediainfo
dvipdf
dwell-click-applet
ecryptfs-add-passphrase
ecryptfsd
ecryptfs-insert-wrapped-passphrase-into-keyring
ecryptfs-manager
ecryptfs-migrate-home
ecryptfs-mount-private
ecryptfs-rewrap-passphrase
ecryptfs-rewrite-file
ecryptfs-setup-private
ecryptfs-setup-swap
ecryptfs-stat
ecryptfs-umount-private
ecryptfs-unwrap-passphrase
ecryptfs-wrap-passphrase
edit
editor
editres
eject
empathy
empathy-accounts
empathy-debugger
enc2xs
enchant
enchant-lsmod
env
envsubst
eog
epmd
eps2eps
eqn
erl
erlc
erl_call
esc-m
escript
esd
esdcat
esdcompat
esdctl
esddsp
esdfilt
esdloop
esdmon
esdplay
esdrec
esdsample
espeak
evince
evince-previewer
evince-thumbnailer
evolution
evolution-addressbook-export
ex
exchange-connector-setup-2.28
expand
expiry
expr
extlinux
factor
faillog
fallocate
fc-cache
fc-cat
fc-list
fc-match
fc-query
fc-scan
file
file-roller
find
find2perl
findsmb
finger
firefox
flock
fmt
fold
font2c
fontconfig-voodoo
fonttosfnt
foo2hiperc
foo2hiperc-wrapper
foo2hp
foo2hp2600-wrapper
foo2lava
foo2lava-wrapper
foo2oak
foo2oak-wrapper
foo2qpdl
foo2qpdl-wrapper
foo2slx
foo2slx-wrapper
foo2xqx
foo2xqx-wrapper
foo2zjs
foo2zjs-icc2ps
foo2zjs-pstops
foo2zjs-wrapper
foomatic-combo-xml
foomatic-compiledb
foomatic-configure
foomatic-datafile
foomatic-perl-data
foomatic-ppdfile
foomatic-ppd-options
foomatic-ppd-to-xml
foomatic-printjob
foomatic-rip
foomatic-searchprinter
free
fribidi
from
fslsfonts
f-spot
f-spot-import
f-spot-sqlite-upgrade
fstobdf
ftp
funzip
gacutil
gacutil2
gamma4scanimage
gawk
gcalctool
gcc
gcc-4.4
gconf-editor
gconf-merge-tree
gconftool
gconftool-2
gcore
gcov
gcov-4.4
gdb
gdbtui
gdebi
gdebi-gtk
gdialog
gdk-pixbuf-query-loaders
gdmflexiserver
gdm-screenshot
gdmsetup
gedit
gencat
genisoimage
geqn
GET
getconf
geteltorito
getent
gethostip
getkeycodes
getopt
gettext
gettext.sh
getweb
ghostscript
ginstall-info
gio-querymodules
gipddecode
gksu
gksudo
gksu-properties
gmenu-simple-editor
gnome-about
gnome-about-me
gnome-appearance-properties
gnome-at-mobility
gnome-at-properties
gnome-at-visual
gnome-audio-profiles-properties
gnome-calculator
gnome-character-map
gnome-codec-install
gnome-control-center
gnome-default-applications-properties
gnome-desktop-item-edit
gnome-dictionary
gnome-display-properties
gnome-doc-prepare
gnome-doc-tool
gnome-file-share-properties
gnome-font-viewer
gnome-help
gnome-keybinding-properties
gnome-keyboard-properties
gnome-keyring
gnome-keyring-daemon
gnome-language-selector
gnome-mouse-properties
gnome-nettool
gnome-network-properties
gnome-open
gnome-panel
gnome-panel-screenshot
gnome-power-manager
gnome-power-preferences
gnome-power-statistics
gnome-screensaver
gnome-screensaver-command
gnome-screensaver-preferences
gnome-screenshot
gnome-search-tool
gnome-session
gnome-session-properties
gnome-session-save
gnome-settings-daemon
gnome-sound-recorder
gnome-system-log
gnome-system-monitor
gnome-terminal
gnome-terminal.wrapper
gnome-text-editor
gnome-thumbnail-font
gnome-typing-monitor
gnome-volume-control
gnome-volume-control-applet
gnome-window-properties
gnome-wm
gnome-www-browser
gpasswd
gpg
gpgsplit
gpgv
gpg-zip
gpic
gprof
groff
grog
grops
grotty
groups
growisofs
grub-bin2h
grub-editenv
grub-fstest
grub-mkelfimage
grub-mkfont
grub-mkimage
grub-mkisofs
grub-mkpasswd-pbkdf2
grub-mkrelpath
grub-mkrescue
grub-script-check
gs
gsbj
gsdj
gsdj500
gslj
gslp
gsnd
gst-feedback-0.10
gst-inspect-0.10
gst-launch-0.10
gstreamer-codec-install
gstreamer-properties
gst-typefind-0.10
gst-visualise-0.10
gst-xmlinspect-0.10
gst-xmllaunch-0.10
gtbl
gtf
gtk-query-immodules-2.0
gtk-update-icon-cache
gtk-window-decorator
gucharmap
gvfs-cat
gvfs-copy
gvfs-info
gvfs-less
gvfs-ls
gvfs-mkdir
gvfs-monitor-dir
gvfs-monitor-file
gvfs-mount
gvfs-move
gvfs-open
gvfs-rename
gvfs-rm
gvfs-save
gvfs-set-attribute
gvfs-trash
gvfs-tree
gwibber
gwibber-accounts
gwibber-error
gwibber-poster
gwibber-preferences
gwibber-service
h2ph
h2xs
hal-device
hal-disable-polling
hal-find-by-capability
hal-find-by-property
hal-get-property
hal-is-caller-locked-out
hal-lock
hal-set-property
hcitool
hd
head
HEAD
helpztags
hexdump
hipercdecode
host
hostid
hp-align
hp-clean
hp-colorcal
hp-fab
hp-firmware
hp-hpdio
hpijs
hp-info
hp-levels
hp-makeuri
hp-mkuri
hp-pkservice
hp-plugin
hp-plugin-ubuntu
hp-probe
hp-query
hp-scan
hp-setup
hp-testpage
hp-timedate
hp-unload
i386
i486-linux-gnu-cpp
i486-linux-gnu-cpp-4.4
i486-linux-gnu-gcc
i486-linux-gnu-gcc-4.4
ibus-daemon
ibus-setup
ibus-table-createdb
iceauth
ico
iconv
id
identify
iecset
igawk
ijs_pxljr
import
im-switch
info
infobrowser
infocmp
infokey
infotocap
inputattach
install
install-info
instmodsh
intel_audio_dump
intel_bios_dumper
intel_bios_reader
intel_error_decode
intel_gpu_dump
intel_gpu_time
intel_gpu_top
intel_gtt
intel_lid
intel_reg_dumper
intel_reg_read
intel_reg_write
intel_stepping
intel_upload_blit_large
intel_upload_blit_large_gtt
intel_upload_blit_large_map
intel_upload_blit_small
invest-chart
ionice
ipcmk
ipcrm
ipcs
ipod-read-sysinfo-extended
ipod-time-sync
iproxy
iptables-xml
isodump
isoinfo
isovfy
ispell-wrapper
jockey-gtk
jockey-text
join
kerneloops-submit
killall
koi8rxterm
kvm-ok
l2ping
last
lastb
lastlog
launchpad-integration
lavadecode
lcf
ld
ld.bfd
ldd
less
lessecho
lessfile
lesskey
lesspipe
lexgrog
lftp
lftpget
libnetcfg
line
link
linux32
linux64
linux-boot-prober
listres
lnstat
loadkeys
loadunimap
locale
localedef
locate
lockfile-check
lockfile-create
lockfile-remove
lockfile-touch
logger
logname
look
lorder
lp
lpoptions
lppasswd
lpq
lpr
lprm
lp_solve
lpstat
lsattr
lsb_release
lscpu
lshal
lshw
lsof
lspci
lspgpot
lss16toppm
lsusb
ltrace
luit
lwp-download
lwp-dump
lwp-mirror
lwp-request
lwp-rget
lxterm
lz
lzcat
lzma
m17n-db
magnifier
mail-lock
mail-touchlock
mail-unlock
make
make-memtest86+-boot-floppy
make_method
mako-render
man
mandb
manhole
manpath
mapscrn
mattrib
mawk
mbadblocks
mcat
mcd
mcheck
mclasserase
mcomp
mcookie
mcopy
mc-tool
mc-wait-for-name
md5sum
md5sum.textutils
mdel
mdeltree
mdir
mdu
mesg
metacity
metacity-message
metacity-theme-viewer
metacity-window-demo
mformat
min12xxw
minfo
mkdiskimage
mkfifo
mkfontdir
mkfontscale
mkisofs
mkmanifest
mk_modmap
mktap
mkzftree
mlabel
mlocate
mmd
mmount
mmove
mogrify
mono
montage
motd+shell
mousetweaks
mpartition
mrd
mren
mscompress
msexpand
mshowfat
mtools
mtoolstest
mtr
mtrace
mtype
mxtar
mzip
namei
nano
nautilus
nautilus-autorun-software
nautilus-connect-server
nautilus-file-management-properties
nautilus-sendto
nawk
ncal
ncurses5-config
ncursesw5-config
neqn
net
netkit-ftp
net.samba3
net-snmp-config
newgrp
ngettext
nice
nl
nm
nm-applet
nmblookup
nmblookup.samba3
nm-connection-editor
nm-tool
nohup
nroff
nslookup
nstat
nsupdate
ntfscat
ntfscluster
ntfscmp
ntfsfix
ntfsinfo
ntfsls
ntfsmount
nvidia-detector
oakdecode
obex-data-server
objcopy
objdump
oclock
od
ods-server
oil-bugreport
oldfind
omshell
on_ac_power
onboard
onboard-settings
oocalc
oodraw
ooffice
oofromtemplate
ooimpress
oomath
ooweb
oowriter
openoffice.org
openssl
opldecode
orca
os-prober
pabrowse
pacat
pacmd
pactl
padsp
pager
palimpsest
pamon
panel-test-applets
paperconf
paplay
parec
parecord
parsechangelog
partx
passwd
paste
pasuspender
pathchk
pax11publish
pcimodules
pcretest
pdb
pdb2.6
pdf2dsc
pdf2ps
pdffonts
pdfimages
pdfinfo
pdfopt
pdftoabw
pdftohtml
pdftoppm
pdftops
pdftotext
peekfd
perl
perl5.10.1
perlbug
perldoc
perlivp
perlthanks
pf2afm
pfbtopfa
pftp
pg
pgawk
pgrep
pic
pico
piconv
pilconvert.py
pildriver.py
pilfile.py
pilfont.py
pilprint.py
pinky
pitivi
pkaction
pkcheck
pkexec
pkg-config
pkill
pl2pm
plog
plymouth-log-viewer
pmap
pm-is-supported
pnm2ppa
pod2html
pod2latex
pod2man
pod2text
pod2usage
podchecker
podselect
poff
pointer-capture-applet
pon
POST
ppmtolss16
pr
precat
preconv
prename
preunzip
prezip
prezip-bin
print
printafm
printenv
printerbanner
printer-profile
printf
protoc
prove
prtstat
ps2ascii
ps2epsi
ps2pdf
ps2pdf12
ps2pdf13
ps2pdf14
ps2pdfwr
ps2ps
ps2ps2
ps2txt
psed
psfaddtable
psfgettable
psfstriptable
psfxtable
pstree
pstree.x11
pstruct
ptar
ptardiff
ptx
pulseaudio
purple-remote
purple-send
purple-send-async
purple-url-handler
pwdx
py3_compilefiles
pycentral
pyclean
pycompile
py_compilefiles
pydoc
pydoc2.6
pygettext
pygettext2.6
pyhtmlizer
python
python2
python2.6
pyversions
qpdldecode
ranlib
rarian-example
rarian-sk-config
rarian-sk-extract
rarian-sk-gen-uuid
rarian-sk-get-cl
rarian-sk-get-content-list
rarian-sk-get-extended-content-list
rarian-sk-get-scripts
rarian-sk-install
rarian-sk-migrate
rarian-sk-preinstall
rarian-sk-rebuild
rarian-sk-update
rcp
rdate
rdesktop
rdfpipe
readelf
readom
red
rename
rename.ul
renice
reset
resize
resizecons
rev
rfcomm
rgrep
rhythmbox
rhythmbox-client
rlogin
routef
routel
rpcclient
rpcgen
rpcinfo
rpl8
rsh
rstart
rstartd
rsync
rtstat
runcon
run_erl
run-mailcap
run-with-aspell
rview
s2p
sane-find-scanner
savelog
scanimage
scp
screen
screendump
screen-launcher
screen-profiles-status
script
scriptreplay
scrollkeeper-config
scrollkeeper-extract
scrollkeeper-gen-seriesid
scrollkeeper-get-cl
scrollkeeper-get-content-list
scrollkeeper-get-extended-content-list
scrollkeeper-get-index-from-docpath
scrollkeeper-get-toc-from-docpath
scrollkeeper-get-toc-from-id
scrollkeeper-install
scrollkeeper-preinstall
scrollkeeper-rebuilddb
scrollkeeper-uninstall
scrollkeeper-update
sctp_darn
sctp_status
sctp_test
sdiff
sdptool
seahorse
seahorse-daemon
see
select-default-iwrap
select-editor
sensible-browser
sensible-editor
sensible-pager
sensors
sensors-conf-convert
seq
service
sessreg
setarch
setkeycodes
setleds
setlogcons
setmetamode
setpci
setsid
setterm
setxkbmap
sftp
sg
sha1sum
sha224sum
sha256sum
sha384sum
sha512sum
shares-admin
shasum
showconsolefont
showfont
showkey
showrgb
shred
shuf
simple-scan
size
skill
slabtop
slogin
slxdecode
smartdimmer
smbcacls
smbclient
smbcquotas
smbget
smbpasswd
smbspool
smbtar
smbtree
smproxy
snice
soelim
soffice
software-center
software-properties-gtk
sort
spd-conf
spd-say
speaker-test
speech-dispatcher
speechd-server-spawn
splain
split
splitfont
sprof
ssh
ssh-add
ssh-agent
ssh-argv0
ssh-askpass
ssh-copy-id
ssh-keygen
ssh-keyscan
ssh-vulnkey
start_embedded
start-pulseaudio-x11
startx
stat
strace
stream
strings
strip
sudo
sudoedit
sum
synclient
syndaemon
syslinux
syslinux2ansi
system-config-printer
system-config-printer-applet
tabs
tac
tail
tap2deb
tap2rpm
tapconvert
tasksel
taskset
tbl
tclsh
tclsh8.4
tee
telnet
telnet.netkit
test
testparm
testparm.samba3
tgz
tic
time
time-admin
tload
toe
tomboy
tomboy-panel
top
toshset
totem
totem-audio-preview
totem-video-indexer
totem-video-thumbnailer
touch
tput
tr
tracepath
tracepath6
traceroute6
traceroute6.iputils
transmission
trial
troff
truncate
tsclient
tset
tsort
tty
twistd
tzselect
u1sdtool
ubiquity
ubiquity-dm
ubiquity-greeter-gtk
ubuntu-bug
ubuntuone-launch
ubuntuone-preferences
ubuntu-support-status
ucf
ucfq
ucfr
ucs2any
udisks
ul
umax_pp
unattended-upgrade
unattended-upgrades
unexpand
unicode_stop
uniq
unlink
unlzma
unopkg
unshare
unzip
unzipsfx
update-alternatives
updatedb
updatedb.mlocate
update-desktop-database
update-gconf-defaults
update-manager
update-mime-database
update-mime-database.real
update-notifier
update-pciids
update-perl-sax-parsers
upower
uptime
usb-creator-gtk
usb-devices
usb_printerid
users
users-admin
user-setup
uuidgen
uxterm
uz
vi
view
viewres
vim.tiny
vinagre
vino-passwd
vino-preferences
vmmouse_detect
vmstat
volname
w
w3m
w3mman
wall
watch
wc
wftopfa
wget
whatis
whereis
which
whiptail
who
whoami
whois
wish
wish8.4
withsctp
wodim
word-list-compress
wpa_passphrase
w.procps
write
www-browser
X
X11
x11perf
x11perfcomp
xargs
xauth
xbiff
xbrlapi
xcalc
xclipboard
xclock
xcmsdb
xconsole
xcursorgen
xcutsel
xdg-desktop-icon
xdg-desktop-menu
xdg-email
xdg-icon-resource
xdg-mime
xdg-open
xdg-screensaver
xdg-user-dir
xdg-user-dirs-gtk-update
xdg-user-dirs-update
xditview
xdpyinfo
xdriinfo
xedit
xev
xeyes
xfd
xfontsel
xfsinfo
xgamma
xgc
xhost
xinit
xinput
xkbbell
xkbcomp
xkbevd
xkbprint
xkbvleds
xkbwatch
xkeystone
xkill
xload
xlogo
xlsatoms
xlsclients
xlsfonts
xmag
xman
xmessage
xml2po
xmlcatalog
xmllint
xmodmap
xmore
Xorg
xpath
xpcshell-1.9.2
xprop
xqxdecode
xrandr
xrdb
xrefresh
xscreensaver-getimage
xscreensaver-getimage-file
xscreensaver-getimage-video
xscreensaver-gl-helper
xscreensaver-text
x-session-manager
xset
xsetmode
xsetpointer
xsetroot
xsetwacom
xsltproc
xsm
xstdcmap
xsubpp
xterm
x-terminal-emulator
xulrunner
xulrunner-1.9.2
xvidtune
xvinfo
xwd
x-window-manager
xwininfo
xwud
x-www-browser
xxd
yelp
yes
zdump
zenity
zip
zipcloak
zipgrep
zipinfo
zipnote
zipsplit
zjsdecode
zsoelim
</pre>
</details>

**Reply by hello-smile6** [reply]
@copy What's a `.debian-installer` file?

**Reply by jdev082** [reply]
@hello-smile6 https://www.debian.org/devel/debian-installer/

**Reply by jdev082** [reply]
sorry that's the wrong thing.

**Reply by hello-smile6** [reply]
> @hello-smile6 https://www.debian.org/devel/debian-installer/

That didn't exactly help, but I discovered https://cdimage.debian.org/cdimage/unofficial/non-free/firmware/ ! Thanks!

**Reply by hello-smile6** [reply]
I'm running with 2 GB of RAM for the guest OS.

**Reply by hello-smile6** [reply]
@JaydenDev How much RAM do you have?

**Reply by jdev082** [reply]
@hello-smile6 12 GB on my broken PC, 4 GB on my current.

**Reply by hello-smile6** [reply]
> @hello-smile6 12 GB on my broken PC, 4 GB on my current.

How's your broken one broken?

**Reply by jdev082** [reply]
@hello-smile6 the cause is not yet known, the issue is mysterious. I can't quite explain it.

**Reply by hello-smile6** [reply]
> @hello-smile6 the cause is not yet known, the issue is mysterious. I can't quite explain it.

Okay.

**Reply by jdev082** [reply]
off topic as well!

**Reply by hello-smile6** [reply]
Just found https://packages.debian.org/sid/elinks

**Reply by hello-smile6** [reply]
> off topic as well!

Sorry

**Reply by hello-smile6** [reply]
@JaydenDev What's the equivalent to <code>chrome://gpuclean</code> on Ubuntu 10?

**Reply by hello-smile6** [reply]
@copy How do I make v86 boot from a `ReadableStream` with async enabled?

**Comment by SuperMaxusa** [comment]
Okay, I tried launch qemu 1.5 (x86_64) in ReactOS v.0.4.14 through v86. Unfortunately, I can't run any image. This works buggy and slowly, but it's works! :D
![image](https://user-images.githubusercontent.com/41739128/148450812-55633112-974f-4792-a097-9ccf78095ec9.png)

**Reply by hello-smile6** [reply]
@SuperMaxusa Try running a 64-bit Floppy Bird build.

**Reply by hello-smile6** [reply]
@Unzor Judging from the network activity, they didn't download it, so they put the exe on a hard disk image and booted from it with v86

**Comment by grapeli** [comment]
You can easily run [dosbox-svn](https://sourceforge.net/projects/dosbox/) and [DOSBox-X](https://github.com/joncampbell123/dosbox-x/).
Download and unpack the image prepared by me.
[copy.sh-linux-dosbox.zip](https://github.com/copy/v86/files/8884877/copy.sh-linux-dosbox.zip)

Load `vmlinuz` as kernel image and initrd as `dosbox.sqf` or `dosbox-x.sqf` and hit start emulation.
After exiting dosbox(-x), if you want to login on the console, press enter or alt-f2.

Don't expect any extraordinary speed. Remember that this is an emulator running in the emulator. Considering this, both work tolerably.

x86 emulation under dosbox is slower than for example under qemu (tcg), but the DOS compatibility is amazing.

**Comment by grapeli** [comment]
Minimal Linux 2.4.36 image with qemu-0.12.5 (static version) and Linux 4.19.254 x86_64.
[qemu.copy.sh.img.zip](https://github.com/copy/v86/files/9285244/qemu.copy.sh.img.zip)

The x86_64 system loads very slowly but in an acceptable time. Although you have to be patient. Load the image as Hard drive disk image. After logging in, press the up arrow and enter.

![copy sh](https://user-images.githubusercontent.com/452325/183521307-f4ea27c8-fb15-420e-8fdf-f739e46a4837.png)

The kernel has no acpi support, so poweroff doesn't work as it should. If you want to quit Qemu, switch to the other console (alt-f2) and type `pkill -QUIT qemu` (you won't do this in graphical mode).
Qemu is built with SDL1. In graphics mode it works only with framebuffer (fbcon). If you run on a system where gpm is not running you must export the `SDL_NOMOUSE=1` environment variable or declare it when starting qemu.

Qemu itself.
[qemu-0.12.5-i686-static.tar.xz.zip](https://github.com/copy/v86/files/9285248/qemu-0.12.5-i686-static.tar.xz.zip)
It should work on most linux x86_32 (64) systems. It's a very old version, but runs much faster under copy.sh than the much newer versions.
